### PR TITLE
Campaign Level Previews Completed

### PR DESCRIPTION
**Pull request by Samuel-Fox**
# Description

Added in screenshots of every campaign level before and after it is completed to be used as thumbnail preview images for the levels.

## Related issue(s)

#68 

## List of changes

A more detailed list of the changes.
- Added screenshots for incomplete and complete levels
- Assigned the appropriate screenshots to each level

### CODE REVIEW

File: Birthday-2024-Project/ScenePrefabs/Pieces/base_piece.tscn [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[gd_scene load_steps=9 format=3 uid="uid://ccdq0btysgv1m"]
+[gd_scene load_steps=9 format=3 uid="uid://cjih77cmvq7fm"]
```
Comment (Casulono) [inline Birthday-2024-Project/ScenePrefabs/Pieces/base_piece.tscn:1]: it's not the uid that other pieces use, it should be ccdq0btysgv1m

File: Birthday-2024-Project/export_presets.cfg [hunk 2]
```diff
@@ -8,7 +8,7 @@ custom_features=""
 export_filter="all_resources"
 include_filter=""
 exclude_filter=""
-export_path="../../Builds/Windows/FaunasOrganizedNeatlyLevelEditor.exe"
```
Comment (Casulono) [inline Birthday-2024-Project/export_presets.cfg:11]: was it an intended change?

**Review by Casulono — CHANGES_REQUESTED** [review]

File: Birthday-2024-Project/export_presets.cfg (hunk 2 above)
Comment (Samuel-Fox) [reply Birthday-2024-Project/export_presets.cfg:11]: With this and the other problem, I did not change anything other than adding files and assigning the screenshots to the levels, I don't know why these other things have changed

Comment (Samuel-Fox) [reply Birthday-2024-Project/export_presets.cfg]: Both problems should be fixed now

File: Birthday-2024-Project/MainScenes/main_level.tscn [hunk 3]
```diff
@@ -218,9 +218,6 @@ controller = NodePath("../GameManager")
 [node name="MainScreen" parent="PuzzleUiManager" index="0" node_paths=PackedStringArray("controller")]
 controller = NodePath("../../GameManager")
 
-[node name="Fade" parent="PuzzleUiManager/MainScreen" index="0"]
```
Comment (Casulono) [inline Birthday-2024-Project/MainScenes/main_level.tscn]: removed a bit too much

**Review by Casulono — APPROVED** [review]

**Pull request merged**